python/nano/src/bigdl/nano/pytorch/vision/datasets/datasets.py

```diff
@@ -62,16 +68,14 @@ def _read_image_to_bytes(self, path: str):
         return img_str
 
     def _decode_img_libjpeg_turbo(self, img_str: str):
-        if self.jpeg is None:
-            self.jpeg = TurboJPEG(lib_path=local_libturbo_path)
         bgr_array = self.jpeg.decode(img_str)
```

Comment:
if self.jpeg is None?

Reply:
before this PR, `self.jpeg` is initialized to `None` on line 56, so line 66 will execute once and only once